test.html

```diff
@@ -166,7 +170,6 @@
             _.delay(callback, 1000, "http://fake.image.url");
         },
         apiOptions: {
-            enableOldAnswerTypes: true,
```

Comment:
I'm going to leave this in, but in the future I'll make small unrelated changes separately.